modules/host/negotiate.go

```diff
@@ -16,6 +16,11 @@ import (
 	"github.com/NebulousLabs/Sia/types"
 )
 
+const (
+	// maxRevisionSize is the maximum number of bytes added in a single revision
+	maxRevisionSize = 100e6 // 100 MB
```

Comment:
100e6 -> one of the first times I actually had trouble figuring out what something said on my small test. Didn't help that the '0's had dots in the middle